mslib/msui/msui_mainwindow.py

```diff
@@ -37,6 +37,8 @@
 import re
 import sys
 import fs
+import json
+import asyncio
 
```

Comment:
in the client asyncio is ok. We have to be careful not to add it to something also the flask socketio server uses.